src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -46,10 +46,14 @@
 
     public static final String VALID_MODULENAME_CS2103T = "CS2103T";
     public static final String VALID_MODULENAME_ES2660 = "ES2660";
+    public static final String VALID_MODULELESSONTYPE_CS2103T = "Lecture";
+    public static final String VALID_MODULELESSONTYPE_ES2660 = "Tutorial";
     public static final String VALID_ZOOMLINK_CS2103T = "www.cs2103tzoom.us";
```

Comment:
I think we might have to change the sample zoom links in this CommandTestUtil class. We should follow the validation regex used to test the validity of zoom links.